reana_job_controller/kubernetes_job_manager.py

```diff
@@ -426,6 +434,64 @@ def _add_voms_proxy_init_container(self, secrets_volume_mount, secret_env_vars):
             voms_proxy_container
         )
 
+    def _add_rucio_init_container(self, secrets_volume_mount, secret_env_vars):
+        """Add sidecar container for a job."""
+        ticket_cache_volume = {"name": "rucio-cache", "emptyDir": {}}
+        volume_mounts = [
+            {
+                "name": ticket_cache_volume["name"],
+                "mountPath": current_app.config["RUCIO_CACHE_LOCATION"],
+            }
+        ]
+
+        rucio_config_file_path = os.path.join(
+            current_app.config["RUCIO_CACHE_LOCATION"],
+            'rucio.cfg',
+        )
+
+        cern_bundle_path = os.path.join(
+            current_app.config["RUCIO_CACHE_LOCATION"],
+            'CERN-bundle.pem',
+        )
+
+        rucio_account = os.environ.get("RUCIO_USERNAME")
+        voms_proxy_vo = os.environ.get("VONAME")
+
+        rucio_config_container = {
+            "image": "reanahub/reana-auth-rucio:latest",
```

Comment:
Can you please move this hardcoded string into `config.py` where we generally assemble all these "magic constants"?

Also, instead of the `latest` tag, please use the tentative `1.0.0` release tag already.  (And for local testing we can simply re-tag it manually on our laptops for development purposes.)  In this way the branch will be ready for merge. 
 We don't use `latest` officially anywhere, because it is a moving target, not identifying individual versions enough.  See also https://vsupalov.com/docker-latest-tag/

If something is static and will never change, we can keep it hard-coded here. If something may change in the future, such as sidecar image version, we should definitely move it to `config.py`. See the VOMS proxy sidecar "magic constants" for illustration: 

```python
VOMSPROXY_CONTAINER_IMAGE = os.getenv(
    "VOMSPROXY_CONTAINER_IMAGE", "reanahub/reana-auth-vomsproxy:1.1.0"
)
"""Default docker image of VOMSPROXY sidecar container."""

VOMSPROXY_CONTAINER_NAME = "voms-proxy"
"""Name of VOMSPROXY sidecar container."""

VOMSPROXY_CERT_CACHE_LOCATION = "/vomsproxy_cache/"
"""Directory of voms-proxy certificate cache.

This directory is shared between job & VOMSPROXY container."""

VOMSPROXY_CERT_CACHE_FILENAME = "x509up_proxy"
"""Name of the voms-proxy certificate cache file."""
```